packages/create-toolpad-app/src/index.ts

```diff
@@ -36,7 +36,7 @@ function getPackageManager(): PackageManager {
       return 'npm';
     }
   }
-  return 'yarn';
+  return 'pnpm';
```

Comment:
The default should probably be `npm` as that is bundled with each node installation (at least for now)